### ISSUE TITLE
Bug: unmet dependency when using typescript 5.9.2

### ISSUE THREAD

**Issue by neilenns** [kinde-oss/kinde-typescript-sdk#79]
### Prerequisites

- [x] I have searched the repository’s issues and [Kinde community](https://thekindecommunity.slack.com/archives/C04K316BXEH) to ensure my issue isn’t a duplicate
- [x] I have checked the latest version of the library to replicate my issue
- [x] I have read the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md)
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md)

### Describe the issue

This project has a dependency on `@typescript-eslint/parser 8.38.0`, which requires typescript `>=4.8.4 <5.9.0`.

I have typescript 5.9.2 installed, however, so I get this warning every time I do `pnpm install`:

```
@kinde-oss/kinde-typescript-sdk 2.12.0
    └─┬ @typescript-eslint/parser 8.38.0
      ├── ✕ unmet peer typescript@">=4.8.4 <5.9.0": found 5.9.2
      └─┬ @typescript-eslint/typescript-estree 8.38.0
        ├── ✕ unmet peer typescript@">=4.8.4 <5.9.0": found 5.9.2
        └─┬ @typescript-eslint/project-service 8.38.0
          ├── ✕ unmet peer typescript@">=4.8.4 <5.9.0": found 5.9.2
          └─┬ @typescript-eslint/tsconfig-utils 8.38.0
            └── ✕ unmet peer typescript@">=4.8.4 <5.9.0": found 5.9.2
```

### Library URL

https://github.com/kinde-oss/kinde-typescript-sdk/

### Library version

2.12.0

### Operating system(s)

Windows

### Operating system version(s)

Windows 11

**Comment by neilenns** [comment]
See https://github.com/typescript-eslint/typescript-eslint/issues/11389, it should be updated this week.

**Comment by pesickaa** [comment]
The Kinde NextJS SDK has been updated to use `@kinde-oss/kinde-typescript-sdk":"2.13.2"`, so this should be fixed. I'm closing this issue, but @neilenns If you still see this issue, please let us know in a new issue.
